src/Services/Cda/CdaTemplateImportDispose.php

```diff
@@ -797,8 +797,11 @@ public function InsertImmunization($imm_array, $pid, CarecoordinationTable $care
             }
             $option = '';
             if (!empty($value['reason_code'])) {
+                // TODO: check with @sjpadgett why the codes are "SNOMED CT:<number>" instead of "SNOMED-CT" for the immunization reason code
+                $reason_code = str_replace("SNOMED CT", "SNOMED-CT", $value['reason_code']);
```

Comment:
Because SNOMED CT is the correct name for the code type. OpenEMR add the dash for the proper reason of search/parse etc. Still Ken handles this in the services and unclear why the current version works for me and not you.
Even still this is an appropriate fix so, yeah!